waku/v2/waku_archive/driver/queue_driver/queue_driver.nim

```diff
@@ -272,29 +268,38 @@ method getMessages*(
   var pageRes: QueueDriverGetPageResult
   try:
     pageRes = driver.getPage(maxPageSize, ascendingOrder, cursor, matchesQuery)
-  except:  # TODO: Fix "BareExcept" warning
+  except CatchableError, Exception:
     return err(getCurrentExceptionMsg())
 
   if pageRes.isErr():
     return err($pageRes.error)
+  else:
+    return ok(pageRes.value)
```

Comment:
A bit nitpicky here, but I prefer having the "default" return case here be the last statement as before and not just a part of the conditional here. That way the last statement is your expected happy-path return and other returns are either fail conditions (or quick successes).